warehouse/query-core/src/main/java/datawave/query/tables/ShardQueryLogic.java

```diff
@@ -165,6 +165,8 @@
      */
     public static final String DATATYPE_FILTER_SET = "datatype.filter.set";
     
+    public static final String DEFAULT_DATE_TYPE_NAME = "default.date.type.name";
```

Comment:
What is the point of this constant?

Reply:
leftover from previous experiment.  removing.